Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -80,6 +85,9 @@ public protocol AppDelegateViewModelOutputs {
   /// Return this value in the delegate method.
   var facebookOpenURLReturnValue: MutableProperty<Bool> { get }
 
+  /// Emits when the view needs to figure out the redirect URL for the emitted URL.
+  var findRedirectUrl: Signal<URL, NoError> { get }
```

Comment:
i hope this output and the `foundRedirectUrl` input are clear.

basically, when the view model captures the email deep link, it asks the view to figure out where it redirects. then once that is found, the view feeds it back to the view model.